scripts/test_e2ee_integration.py

```
@@ -4,58 +4,55 @@
 
-    # Create dummy classes to prevent further import errors
-    class _FallbackE2EEDebugUtilities:
-        pass
 
+class MatrixConnectionError(Exception):
+    """Raised when Matrix connection fails."""
 
-E2EEDebugUtilities = (
-    _ImportedE2EEDebugUtilities if IMPORTS_AVAILABLE else _FallbackE2EEDebugUtilities
-)
+    pass
```

Ruff (0.15.12)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)